docs/IntegrationWithExistingApps.md

```diff
@@ -248,6 +248,30 @@ end
 
 <block class="objc swift" />
 
+> **Note:** From React Native `0.42.0` you have to explicitly include Yoga in your Podfile. Here is an example:
+
+```
+source 'https://github.com/CocoaPods/Specs.git'
+
+# Required for Swift apps
+platform :ios, '8.0'
+use_frameworks!
+
+# The target name is most likely the name of your project.
+target 'your-project' do
+
+  react_native_path = "../node_modules/react-native"
+  pod 'React', :path => react_native_path, :subspecs => [
+    'Core',
+    'RCTText',
+    'RCTNetwork',
+    'RCTWebSocket',
+  ]
+  # Explicitly include Yoga
```

Comment:
Just adding the line with a comment that says it only applies to RN >= 0.42.0 should be sufficient. A whole new example might add too much noise and cause the opposite effect of people glancing over it.